bundles/org.openhab.binding.bmwconnecteddrive/README.md

```diff
@@ -0,0 +1,686 @@
+| Overall Window Status     | windows             | String        | Combined status for all windows                |
+| Doors Locked              | lock                | String        | Status if doors are locked or unlocked         |
+| Next Service Date         | service-date        | DateTime      | Date of upcoming service                       |
+| Mileage till Next Service | service-mileage     | Number:Length | Mileage till upcoming service                  |
+| Check Control             | check-control       | String        | Presence of active warning messages            |
+| Charging Status           | charge              | String        | Only available for phev, bev_rex and bev       |
+| Last Status Timestamp     | last-update         | DateTime      | Date and time of last status update            |
+
+Overall Door Status values
+
+* _Closed_ - all doors closed
+* _Open_ - at least one door is open
+* _Unknown_ - no door data delivered at all
+
+Overall Windows Status values
+
+* _Closed_ - all windows closed
+* _Open_ - at least one window is completely open
+* _Intermediate_ - at least one window is partially open
+* _Unknown_ - no window data delivered at all
+
+Check Control values
+
+* _Active_ - at least one warning message is active
+* _Not Active_ - no warning message is active
+* _Unknown_ - no data for warnings delivered
+
+Charging Status values
+
+* _Charging_
+* _Error_
+* _Finished Fully Charged_
+* _Finished Not Full_
+* _Invalid_
+* _Not Charging_
+* _Charging Goal reached_
+* _Waiting For Charging_
+
+#### Services
+
+Group for all upcoming services with description, service date and/or service mileage.
+If more than one service is scheduled in the future the channel _name_ contains all future services as options.
+
+* Channel Group ID is **service**
+* Available for all vehicles
+* Read/Write access
+
+| Channel Label                  | Channel ID          | Type           | Access     |
+|--------------------------------|---------------------|----------------|------------|
+| Service Name                   | name                | String         | Read/Write |
+| Service Date                   | date                | Number         | Read       |
+| Mileage till Service           | mileage             | Number:Length  | Read       |
+
+#### Check Control
+
+Group for all current active CheckControl messages.
+If more than one message is active the channel _name_ contains all active messages as options.
+
+* Channel Group ID is **check**
+* Available for all vehicles
+* Read/Write access
+
+| Channel Label                   | Channel ID          | Type           | Access     |
+|---------------------------------|---------------------|----------------|------------|
+| CheckControl Description        | name                | String         | Read/Write |
+| CheckControl Mileage Occurrence | mileage             | Number:Length  | Read       |
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
Corrected